client/src/pages/LandingPage.js

```diff
@@ -1,11 +1,31 @@
+
 import React from 'react';
-import Header from '../components/Header/Header';
+import HeaderLP from '../components/Header/HeaderLP'
```

Comment:
We should use the Header file that already exists.  This contains current Login logic that works.  We should remove the HeaderLP file.

Reply:
Check update